bigint/src/bigint.rs

```diff
@@ -926,6 +926,56 @@ impl BigInt {
         BigInt::from_biguint(sign, BigUint::new(digits))
     }
 
+    /// Creates and initializes a `BigInt` from an array of bytes in two's complement.
+    ///
+    /// The digits are in little-endian base 2^8.
+    #[inline]
+    pub fn from_twos_complement_bytes_le(mut digits: Vec<u8>) -> BigInt {
```

Comment:
I do think this should take `&[u8]`, even if that means making extra copies for now.  We can work on optimizing that later, but the public interface needs to be stable.